src/VisualStudio/Xaml/Impl/Features/Completion/XamlCommitCharacter.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
```

Comment:
Why?

Reply:
hmm.... I didn't add that. I believe it is autogenerated and I didn't pay attention to what have been added usually...

---
In reply to: [618733501](https://github.com/dotnet/roslyn/pull/52848#discussion_r618733501) [](ancestors = 618733501)

Reply:
Yeah, I guess I don't understand how nullable works in mixed scenarios.

---
In reply to: [618738797](https://github.com/dotnet/roslyn/pull/52848#discussion_r618738797) [](ancestors = 618738797,618733501)

Reply:
I think you should be able to remove this.  We may want to look at changing our defaults